app/main.cpp

```diff
@@ -158,7 +160,8 @@ int main(int argc, char *argv[])
     }
 
     QCommandLineParser parser;
-    parser.setApplicationDescription(QStringLiteral("Perf data parser and unwinder."));
+    auto elfutilsVersion = QString::fromUtf8(dwfl_version(nullptr));
+    parser.setApplicationDescription(QStringLiteral("Perf data parser and unwinder.\nBuild with elfutils version: %1").arg(elfutilsVersion));
```

Comment:
Not "Build with" - "Using". You could also add the "build with" in addition if you want, doubt it's useful